packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -1,7 +1,7 @@
 // We allow dead-code here, because some of the functions are only exposed when `feature = "hooks"`
 // we could do cfg for everything, but that gets very messy, instead we only use a subset
 // and enable deadcode on `feature = "std"`.
-#![cfg_attr(not(feature = "std"), allow(dead_code))]
+#![cfg_attr(not(feature = "std"), allow(dead_code, unreachable_pub))]
```

Comment:
See above

Reply:
fixed in [3ca638d](https://github.com/hashintel/hash/pull/975/commits/3ca638d6f7029b2f91e046ac6badb66e28be2142)